pom.xml

```diff
@@ -261,7 +261,7 @@
     <version.org.wildfly>18.0.1.Final</version.org.wildfly>
     <version.org.wildfly.arquillian>2.2.0.Final</version.org.wildfly.arquillian>
     <version.org.wildfly.core>10.0.3.Final</version.org.wildfly.core>
-    <version.org.wildfly.common>1.5.1.Final</version.org.wildfly.common>
+    <version.org.wildfly.common>1.5.4.Final-format-001</version.org.wildfly.common>
```

Comment:
this could be controversial upgrade due the version is that really valid thing due our versioning here in JBoss MW?

Reply:
Good point. Shall I change it to 1.5.4.Final?